Cargo.toml

```
@@ -70,6 +73,11 @@ memoffset = "0.9"
 crevice = "0.13"
 typed-arena = "2.0"
 ordered-float = "3.3"
+percent-encoding = { version = "2.3.0", optional = true }
+base64 = { version = "0.21.2", optional = true }
+gltf = { version = "1.2.0", optional = true }
```

```suggestion
gltf = { version = "1.2.0", optional = true, default-features = false }
```

I don't think that we really need these